### ISSUE TITLE
Maybe FTB Quests?

### ISSUE THREAD

**Issue by TheAirBlow** [Nomi-CEu/Nomi-CEu#272]
A little bit personal, but:
1) Is less wonky that BQ (for example it renders outside the book)
2) IMHO looks a lot better and fits the realm of tech more than an old book
3) Better UX/UI and QoL for editing, but sadly doesn't support quest book translation (can be fixed with a humble & simple fork)
4) Doesn't have weird syncing behaviour (BQ doesn't sync checkbox quests, just an annoyance but still)
5) One team for everything, and it's a lot advanced than BQ parties
6) Allows to view quests before unlocking

Half the job can be done with the converter mod, but sadly a lot of the work has to be done manually because of the difference between those two.

May not be worth it from dev's perspective, but will remove all annoyances for the players.

**Comment by IntegerLimit** [comment]
This has been discussed a few times, both here, and in the nomi discord, for both Nomifactory and Nomifactory CEu. The problem with FTB Quests is that it shows large blobs of text, which is the whole nomi qb, much worse than BQu.

Furthermore, at the moment, I am more focused on other stuff (like getting rid of activated blocks), so this is not too high on the priority (to-consider) list. I am leaving this open for now, to be further considered in the future, but the current answer is no.

**Comment by IntegerLimit** [comment]
Honestly, might happen, with patchouli. Re-opening.

**Comment by tracer4b** [comment]
FTB quests has issues with GT packs due to performance issues with larger questbooks and with long quest dependency chains. We can't make it work for Nomi

**Comment by TheAirBlow** [comment]
> FTB quests has issues with GT packs due to performance issues with larger questbooks and with long quest dependency chains. We can't make it work for Nomi

Doesn't Technological Journey use FTB Quests? It is quite large.

**Comment by IntegerLimit** [comment]
Yeah honestly its fine. Main problem is showing large amounts of text on a quest, but that is fixed with patchaholi.

**Comment by TheAirBlow** [comment]
> Yeah honestly its fine. Main problem is showing large amounts of text on a quest, but that is fixed with patchaholi.

So basically usinf patchouli for documentation? And not like we can't make a mixin mod to make it scale horizontally too (but it is more difficult bc forking is not allowed, fuck FTB)

**Comment by IntegerLimit** [comment]
Removed questbook label, as questbook label is for questbook issues, not stuff related to the infra of the questbook.
